docs/data-sources/parameter.md

```diff
@@ -56,6 +56,7 @@ Optional:
 - `error` (String) An error message to display if the value doesn't match the provided regex.
 - `max` (Number) The maximum of a number parameter.
 - `min` (Number) The minimum of a number parameter.
+- `monotonic` (String) Number monotonicity, either increasing or decreasing.
```

Comment:
👍 

nit: we may want to improve the Makefile to detect discrepancies.

Reply:
This is done via `make gen`, we should probably verify in CI that there are no changes.

Reply:
Yes, that's what I meant.

Reply:
I'll toss up a PR 👍🏻